packages/frontend/src/App.svelte

```diff
@@ -110,6 +111,11 @@ onDestroy(() => {
           <Preferences />
         </Route>
 
+        <!-- AI Lab Service -->
+        <Route path="/ai_lab_service">
+          <AiLabService />
+        </Route>
+
```

Comment:
question: why name it AI Lab service (the AI lab prefix ?), as it's already part of AI lab, needs to only be /service 

The same for AiLabService (need to remove the prefix)

Reply:
anyway, as it's within a section named ServerInformation , then it should be `ServerInformation` and route path `/server-information`